### PR TITLE
[Docs] Add inferno-enzyme-adapter to README.md

### PR DESCRIPTION
**Pull request by eddyerburgh**
- Add inferno-enzyme-adapter to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ to make a PR to this README and add a link to it! The known 3rd party adapters a
 | Adapter Package | For Library | Status |
 | --- | --- | --- |
 | [`preact-enzyme-adapter`](https://github.com/aweary/preact-enzyme-adapater) | [`preact`](https://github.com/developit/preact) | (work in progress) |
+|[`inferno-enzyme-adapter`](https://github.com/bbc/inferno-enzyme-adapter)|[`inferno`](https://github.com/infernojs/inferno)|(work in progress)|
```
Comment (ljharb) [inline README.md]: Why aren't both these named `enzyme-adapter-*`, to mirror the react ones?

(see https://github.com/aweary/preact-enzyme-adapter/issues/5)

Comment (eddyerburgh) [reply README.md]: Agreed, I'll change inferno-enzyme-adapter to enzyme-adapter-inferno.

Comment (ljharb) [reply README.md]: Thanks :-) obviously you can name it whatever you want, but I think it's useful and helpful to set a strong precedent of a naming convention.

Comment (eddyerburgh) [reply README.md]: I've renamed the inferno adapter to enzyme-adapter-inferno

Comment (eddyerburgh) [reply README.md]: Yes agreed

**Comment by eddyerburgh** [comment]
OK, I've rebased into one commit 😀

**Pull request merged**